src/Plotly.NET.Interactive/Repack.ps1

```diff
@@ -1,9 +1,10 @@
-# clean up the previously-cached NuGet packages
-Remove-Item -Recurse ~\.nuget\packages\Plotly.NET.Interactive* -Force
-Remove-Item -Recurse ~\.nuget\packages\Plotly.NET* -Force
+# Clean up the previously-cached NuGet packages.
+# Lower-case is intentional (that's how nuget stores those packages).
```

Comment:
Oh btw, that's a fix of my own mistake. It does not clean well because the packages are stored by nuget in lower-case, which makes a different for pwsh, apparently